dev/format_python_code.sh

```diff
@@ -6,10 +6,10 @@ PYTHON="${PYTHON:-python3}"
 BLACK="${PYTHON} -m black"
 ISORT="${PYTHON} -m isort"
 
-${BLACK} --version >/dev/null 2>&1 && {
+[ ${BLACK} --version >/dev/null 2>&1 ] && {
```

Comment:
This doesn't look right. You can't just put arbitrary commands in `[ ]`:

```console
$ [ python -m black --version ]
-bash: [: syntax error: `-m' unexpected
```

Reply:
Does any of the versions work in your case?

Reply:
No, but that's because the operator is wrong. It should be `||`, not `&&`.

Although for clarity, I would advise to rewrite it as:

```bash
if ! ${BLACK} --version >/dev/null 2>&1; then
    ....
fi
```

Reply:
> It should be ||

Ah, yes. Thanks. Done.